Sources/Layoutable.swift

```diff
@@ -29,6 +29,8 @@ public protocol Layoutable: AnyObject, Equatable, CustomDebugStringConvertible {
     var superview: PinView? { get }
     var subviews: [PinView] { get }
 
+    var isVisible: Bool { get }
```

Comment:
Also, I agree with @MontakOleg, we should avoid adding properties that could conflict with user's possible extension. I like the idea of using the name `isConsideredVisibleForViewFilters`